util/helpers.ts

```diff
@@ -7,12 +7,18 @@ const config = {
   confirmationsForDeploy: 1
 }
 
-export async function deployStablecoin(deployer: Signer, supply: number | string, precision: number | string, confirmations: number = config.confirmationsForDeploy): Promise<Contract> {
+function log(message: string, opts?: { logOutput: boolean }) {
+  if (!opts || opts?.logOutput === true) {
+    console.log(message);
+  }
+}
```

Comment:
This function is defined in two files: this one and `campaign-deployer.ts`. Maybe extract it to `utils/utils.ts`.